src/GraphManager/utils.test.ts

```diff
@@ -264,3 +264,15 @@ describe("linkPointerAreaPaint", () => {
     expect(ctx.stroke).not.toHaveBeenCalled();
   });
 });
+
+import { onGraphUpdate } from "./GraphRenderer";
+
+describe("onGraphUpdate", () => {
+  it("should update the graph data", () => {
+    const ctrl = makeMockController();
+    const graphData = { nodes: [], links: [] };
+    const state = makeGraphState(graphData, jest.fn());
+    onGraphUpdate(ctrl, { graph: { nodes: [], links: [] } }, state.setGraph); // BROKEN
+    expect(ctrl.graph.current).toEqual(graphData);
```

Comment:
Btw. please also add an expectation on the `setGraph` function, something like `expect(..).tohavebeencalled`
This is testing the first call to `convertAndSetGraph`, which was previously untested, so you'd increase our test coverage by doing so! :)